_includes/01-name.md

```diff
@@ -0,0 +1 @@
+Replace this with your name, and include a header.
```

Comment:
# This is my new H1 Title
## with an H2 title for reference
###### and an H6 just for fun

Reply:
Playing around with titles